src/components/footer.tsx

```diff
@@ -86,41 +86,152 @@ export function Footer() {
+                    <FormItem>
+                      <FormLabel className="text-neutral-300">E-mail</FormLabel>
+                      <FormControl>
+                        <Input
+                          className="text-neutral-300 border-muted-foreground focus:border-brand-orange"
+                          placeholder="usuario@dominio.com"
+                          type="email"
+                          {...field}
+                        />
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name="subject"
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel className="text-neutral-300">
+                        Assunto
+                      </FormLabel>
+                      <FormControl>
+                        <Input
+                          className="text-neutral-300 border-muted-foreground focus:border-brand-orange"
+                          placeholder="Digite o assunto"
+                          type="string"
+                          {...field}
+                        />
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name="message"
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel className="text-neutral-300">
+                        Mensagem
+                      </FormLabel>
+                      <FormControl>
+                        <Textarea
+                          placeholder="Digite sua mensagem..."
+                          className="text-neutral-300 border-muted-foreground focus:border-brand-orange resize-none leading-relaxed overflow-hidden"
+                          {...field}
+                        ></Textarea>
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  )}
+                />
+                <Button
+                  type="submit"
+                  className="w-full mt-3 text-md focus:bg-brand-orange hover:bg-brand-orange hover:text-neutral-900 focus:text-neutral-900"
+                >
+                  Enviar
+                </Button>
+              </form>
+            </Form>
+          </div>
+        </div>
+        <div
+          className="w-full sm:w-1/4 p-4 flex flex-col justify-between items-center sm:items-end gap-4 sm:gap-0"
+          id="contato"
+        >
+          <p className="text-muted text-center sm:text-end font-bold text-2xl mb-2">
+            Fale conosco
           </p>
-          <div className="flex gap-4">
+          <p className="text-muted m-0 p-0 text-center sm:text-end">
```

Comment:
Usar <br/> não é la uma boa prática.. Tinha esse costume mas comecei a trabalhar mais com outras alternativas, uma sugestão
```suggestion
          <p className="text-muted m-0 p-0 text-center sm:text-end flex flex-col gap-4">
                      <span>Horário de Funcionamento:</span>
                      <span>Segunda a sexta</span>
                      <span>7h às 17h</span>
                      //.... com o resto das coisas
     
            <Link
              href="https://www.facebook.com/frankmetalicas"
              href="mailto:frank@frankmetalicas.com"
              target="_blank"
              rel="noopener noreferrer"
              className="text-muted hover:text-brand-orange"
            >
              <AiFillFacebook
                className="text-neutral-200 hover:text-brand-orange transition-transform transform hover:scale-110"
                size={30}
              />
              frank@frankmetalicas.com
            </Link>
          </p>
          
```

No caso usa span e transforma em flex e flex col, ai pra espaçamento usa um gap pra determinar, fica mais semântico e evita o uso da tag noggers